### ISSUE TITLE
update to a14 source 

### ISSUE THREAD

**Issue by xxmcvapourxx** [minimal-manifest-twrp/platform_manifest_twrp_aosp#45]
Hi. 
   I'm trying to build twrp source dm3q for s23u. Would you be able to update your source to a14? Or is there a way I can do it? 

Thanks

**Comment by fluffball3** [comment]
this manifest shud build android 12.1+ because my phone still has an android 12 kernel despite being a14 os

**Comment by xxmcvapourxx** [comment]
Not really... there an issue with a glitch on this build within twrp. Using a13 kernel and a14 kernel I've tried on s23u. The glitch doesn't happen on s23 or s23+.. only s23u.

**Comment by fluffball3** [comment]
must be something wrong with the kernel source or the kernel.... what does it say??

**Comment by xxmcvapourxx** [comment]
it's a glitch in twrp. It doesn't say anything. Its a display issue all s23u does it.. No it's not kernel source I've tried a13 and a14 kernel source building twrp. So it's something within twrp that's causing it. 🤔

**Comment by fluffball3** [comment]
if u managed to compile the twrp but facing display errors u shud try modifying boardconfig to add more twrp flags. u can refer to a similar model phone with same chipset or find a prebuilt twrp for ur device model and get the device tree and check its board config there

this might help but its for snapdragon chip:
https://github.com/samsung-sm8550/device_samsung_dm3q/tree/android-12.1

from this:
https://xdaforums.com/t/recovery-unofficial-twrp-for-galaxy-s23-ultra-snapdragon-series.4648638/

**Comment by xxmcvapourxx** [comment]
Dude. Really... Afaneh92 who originally built s21 and s22 and s23 twrp also faced this issue. But he stopped development. So now its open source. He built twrp before even s23u even came out and still yet had display issues with it. And also few other devs have looked. So no it's not board config. It's something within twrp needs to be updated.

**Comment by xxmcvapourxx** [comment]
Thou links lol I have used also still fact he also expecting the glitch lol 😆 😂

**Comment by fluffball3** [comment]
i dont work with those phone models so i was trying to help if ure sure its a bug in twrp then it is i guesss.... still curious tho what does it say like is it just a blank screen?

**Comment by xxmcvapourxx** [comment]
No blank screen. When you scroll or do the combo to get into twrp. It looks like the old 80's TV going fuzzy. That's how I can tell you lol 😆

**Comment by fluffball3** [comment]
if u still wanna try i guess u can try omni sources or lineago os sources since theyre uptodate almost and then get the a14 twrp source too but last time i tried that there were a bunch of errors that i dunno how to solve...

**Comment by xxmcvapourxx** [comment]
That's the thing there no a14 twrp source for my device yet. Hence why I'm asking lol 😆 and yes I'm also using Los source also. 😅

**Comment by fluffball3** [comment]
u can just replace the recovery from lineage with twrp right or am i missiing something?? did u mean device tree?? also twrp sources are more uptodate in android 12.1 than in a14 branch

**Comment by xxmcvapourxx** [comment]
Los is different as its for asop. Twrp is for one ui which we use to flash custom roms. 

Yeah device tree needs to be updated to a14.
